toqito/helper/bell_npa_constraints.py

```diff
@@ -0,0 +1,159 @@
+"""Generates NPA hierarchy constraints for Bell inequalities."""
+
+from typing import List, Tuple, Union
```

Comment:
Assuming the rest of the file uses native types `list`, `tuple`, and `|` over `List`, `Tuple`, and `Union`, we can remove this import.